src/families/bitcoin/ScreenEditCustomFees.js

```diff
@@ -0,0 +1,140 @@
+/* @flow */
+import { BigNumber } from "bignumber.js";
+import invariant from "invariant";
+import React, { useState, useCallback } from "react";
+import { useTranslation, Trans } from "react-i18next";
+import { Keyboard, StyleSheet, TextInput, View } from "react-native";
+import SafeAreaView from "react-native-safe-area-view";
+import { useTheme } from "@react-navigation/native";
+import { getAccountBridge } from "@ledgerhq/live-common/lib/bridge";
+import { useSelector } from "react-redux";
+import type { Transaction } from "@ledgerhq/live-common/lib/families/bitcoin/types";
+import { i18n } from "../../context/Locale";
+import Button from "../../components/Button";
+import KeyboardView from "../../components/KeyboardView";
+import NavigationScrollView from "../../components/NavigationScrollView";
+import LText from "../../components/LText";
+import { accountScreenSelector } from "../../reducers/accounts";
+
+const forceInset = { bottom: "always" };
+
+const options = {
+  title: i18n.t("send.summary.fees"),
```

Comment:
use the <Trans /> component instead unless it is required for it to be a string. also its better to declare these options where you import the screen in your navigator

Reply:
Not sure to understand about this one,
should I add the option title here -> https://github.com/LedgerHQ/ledger-live-mobile/blob/cb089c8c631b97332197dd6fdc8131021ce3a3b8/src/families/bitcoin/SendRowsFee.js#L58

Reply:
ho sorry for the component use <Trans> instead if possible as for the options you can actually keep it here since these components are loaded from the families its good as is :+1: